lib/ae_mdw/channels.ex

```diff
@@ -62,15 +64,17 @@ defmodule AeMdw.Channels do
   end
 
   @spec fetch_channel(state(), binary()) :: {:ok, channel()} | {:error, Error.t()}
```

Comment:
We should change signature here to be `(state(), pubkey())`

Reply:
done